api/main.py

```diff
@@ -86,6 +87,12 @@ class LogConfig(BaseModel):
 
 app.mount("/", SPAStaticFiles(directory="iqengine", html=True), name="iqengine")
 
+# Import all configuration from the environment variables
+try:
```

Comment:
Do we really want to do a catch all rather than being more descriptive about which env variable it will be?